gbpservice/nfp/configurator/drivers/firewall/vyos/vyos_fw_driver.py

```diff
@@ -121,6 +189,26 @@ def configure_interfaces(self, context, resource_data):
 
         """
 
+        mgmt_ip = resource_data['mgmt_ip']
```

Comment:
Log forwarding will be invoked only after HM as this is wrapped inside configure interfaces API.